Black-Market/Black-Market/Components/MarketTexfield.swift

```diff
@@ -30,32 +33,48 @@ struct MarketTexfield: TextFieldStyle {
     
     init(
         style: Binding<Style> = .constant(.default),
-        title: String? = nil
+        title: String? = nil,
+        placeholderText: String = "",
+        fieldValue: Binding<String> = .constant("")
     ) {
         self._style = style
         self.title = title
+        self.placeholderText = placeholderText
+        self._fieldValue = fieldValue
     }
     
-    func _body(configuration: TextField<_Label>) -> some View {
-        
+    func _body(configuration: TextField<Self._Label>) -> some View {
         VStack(alignment: .leading) {
             if let title = title {
-                Text(title).font(.custom("Open Sans", size: UIConstants.MarketTexfield.size))
+                Text(title)
+                    .font(.custom("Open Sans", size: 14))
```

Comment:
I would move the Font styles and sizes to constants or a design system in the app, so they can be reused and the UI is consistent

Reply:
Done